pages/howto/matlab.md

```diff
@@ -0,0 +1,498 @@
+
+>> job.fetchOutputs{:}
+
+ans =
+
+    8.4850
+
+>> 
+
+```
+
+The job ran in 8.485 seconds using four workers.  **Note** that these jobs will always request N+1 CPU cores, since one worker is required to manage the batch job and pool of workers. 
+For example, a job that needs eight workers will request nine CPU cores.
+
+### Configuring Slurm Integration on CCR MATLAB (GUI)
+
+Configuring the Slurm integration using the MATLAB GUI is pretty straight forward. 
+
+- Start up MATLAB through an Ondemand Session as described [here](#running-matlab-gui-though-openondemand)
+
+- From the MATLAB Session Click on `Parallel` -> `Discover Clusters`
+
+- Leave the check box in the `On your Network` and click `Next`
+
+- Highlight the default `ub-hpc` clister and click `Finish`
+
+![](../images/matlab2.png)
+
+You now have a default Cluster Profile that should work out of the box, but you will want to change some of the default settings such as user email settings.
+
+To make changes to the default Cluster Profile:
+
+
+- From the MATLAB Session Click on `Parallel` -> `Create and Manage Clusters`
+
+- Click on the ub-hpc Cluster Profile and Click `Edit`
+
+- Make your changes and click `Done`
+
+
+### Configuring Remote Slurm Integration (GUI)
+
+It is possible to configure your local MATLAB to submit jobs to CCR from your Desktop or Workstation. The setup is very similar to Configuring Slurm Integration on CCR MATLAB once you have the correct integration scripts.
+
+
+NOTE: This proceedure assumes that you have MATLAB R2023b (Currently the only supported Version at CCR) installed on your local machine.
+
+!!! Warning "SSH Key Credentials"
+    Submission to the cluster requires SSH credentials.  You will be prompted for username and password or identity file (private key).  The username and location of the private key will be stored in MATLAB for future sessions.
+
+
+You will need do download the MATLAB CCR Slurm integration script and place them on your local machine in the MATLAB
+
+Download the [CCR Matlab Integration Scripts](https://g-ac407a.8c185.08cc.data.globus.org/R2023b/University-at-Buffalo.zip) and start MATLAB.  The ZIP file should be unzipped in the location returned by calling…
+
+```
+>> userpath
+
+ans =
+
+    '/home/testuser/Documents/MATLAB'
+
+>> 
+
+```
+
+
+Configure MATLAB to run parallel jobs on the cluster by using the configCluster This only needs to be run  once per version of MATLAB.
+
+
+```
+>> configCluster
+Username on UB-HPC (e.g. jdoe): >>testuser
+Complete.  Default cluster profile set to "ub-hpc R2023b".
+>> 
+
+```
+You will need to configure your user infomation into the Profile including the location to your public SSH key:
+
+```
+>> c = parcluster;
+>> c.AdditionalProperties.EmailAddress = 'testuser@buffalo.edu'
+>> c.AdditionalProperties.IdentityFile = '/home/testuser/.ssh/private.key'
+```
+
+**NOTE:** We recommend setting the RemoteJobStorageLocation to your project space rather than your home directory.
```

Comment:
```suggestion
**NOTE:** We recommend setting the RemoteJobStorageLocation to your group's project directory, if you have one, rather than your home directory to prevent going over quota in your home directory.  
```